gr-qtgui/grc/qtgui_range.block.yml

```diff
@@ -53,19 +53,29 @@ parameters:
     dtype: int
     default: '200'
     hide: part
+-   id: outputmsgname
```

Comment:
You should add the option to hide the msg outputport

Reply:
I'm not sure I understand the guidance here. Do you mean it should be `hide: all`? Under which circumstances?

Reply:
Have a look at the qtgui sink:
https://github.com/gnuradio/gnuradio/blob/f44384ae1f061f257603b02288aef4db1b74d4be/gr-qtgui/grc/qtgui_freq_sink_x.block.yml#L116-L120

and 
https://github.com/gnuradio/gnuradio/blob/f44384ae1f061f257603b02288aef4db1b74d4be/gr-qtgui/grc/qtgui_freq_sink_x.block.yml#L380-L398

Even better than using a boll value it would be to use enums, so you could toggle this option instead of entering True or False

Reply:
I believe the PR as-amended now takes this into account.

Reply:
Principally this looks good. Did you try to hide and show the message ports, when there was a an entry in the gui_hint field, for instance 0,0,1,1 ?
 I noticed sometimes some error messages, when the gui_hint parameter appeared befoe the showports parameter.

See #5665